src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/AuthorizationServer/ApplicationController.php

```diff
@@ -67,70 +59,28 @@ public function indexAction(AuthorizedApplicationsFilters $filters): Response
         ]);
     }
 
-    /**
-     * @param Request $request
-     *
-     * @return Response
-     */
-    public function createAction(Request $request): Response
+    public function viewAction(): void
```

Comment:
@mflasquin seems like it's removing some implemented actions I'm not sure about this I guess the actions were finished and merged in develop after the 8.1.x branch was created, but they shouldn't be removed right?

Reply:
that's what it looks like, I should probably put it back in develop, wdyt @mflasquin  ?

Reply:
fixed